recipes/catch2/2.x.x/conanfile.py

```diff
@@ -19,12 +19,14 @@ class ConanRecipe(ConanFile):
         "with_main": [True, False],
         "with_benchmark": [True, False],
         "with_prefix": [True, False],
+        "default_reporter": "ANY",
```

Comment:
I think it can only gain a limited set of values? https://github.com/catchorg/Catch2/blob/103cb16696b7972e4f53aacea87f522ba63d2d31/src/catch2/reporters/catch_reporters_all.hpp#L24

Reply:
Yes, but this list might change with versions, and might be considered as a sort of "implementation detail". So I chose not to specify it directly.

Reply:
Do you want me to put the list explicitly?

Reply:
nope, just wanted to clarify it, for now `ANY` is okay